packages/internal/src/typeGuards.js

```diff
@@ -4,9 +4,22 @@
 import { M } from '@endo/patterns';
 
 /**
+ * @import {DataOnly, RemotableBrand} from '@endo/eventual-send';
  * @import {TypedPattern} from './types.js';
  */
 
+/**
+ * Mark a value with RemotableBrand.
+ *
+ * @template T
+ * @param {T} value
+ * @returns {T & RemotableBrand<DataOnly<T>, T>}
+ */
+export const remotable = value =>
```

Comment:
`asRemotable` might be more clear.

I once did a noop function like this to factor out typecasts and I got pushback that persuaded me to inline the cast. But it only occurred in 2 or 3 places. Perhaps we're using this one a lot?

Reply:
I oppose runtime calls to perform a type cast. @michaelfig says it's onerous to do in JSDoc. I'm not fully convinced but I won't block on it. I would like to see where this is used because it's not in this PR. At those usage sights I'd like to understand why an unchecked cast is legitimate.

I will block on the function name to make clear that all it's doing is a cast. So please rename. `asRemotable` sounds like it's actually converting the value so I suggest,

```suggestion
export const castRemotable = value =>
```

And update the JSDoc to indicate that it's not affecting the runtime in any way.

Reply:
Oh, I intended to revert this change (since it was not sufficiently motivated)... must've gotten lost in my rebasing.  It'll be gone in the next push.